selenium_webdriver/pom.xml

```diff
@@ -3,15 +3,76 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
     <groupId>org.example</groupId>
     <artifactId>selenium_webdriver</artifactId>
     <version>1.0-SNAPSHOT</version>
-
     <properties>
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <testng.version>7.4.0</testng.version>
+        <log4j.version>1.2.17</log4j.version>
+        <maven-surefire-plugin.version>2.22.1</maven-surefire-plugin.version>
+        <selenium-java.version>4.7.2</selenium-java.version>
+        <webdrivermanager.version>5.3.1</webdrivermanager.version>
+        <slf4j-simple.version>1.6.1</slf4j-simple.version>
+        <asertj-core.version>3.24.1</asertj-core.version>
+        <lombok.version>1.18.20</lombok.version>
```

Comment:
please add empty line to separate properties with version value

Reply:
Fixed